src/sentry/static/sentry/app/views/organizationIncidents/details/body.jsx

```diff
@@ -16,11 +16,21 @@ import theme from 'app/utils/theme';
 import Activity from './activity';
 import IncidentsSuspects from './suspects';
 import detectedSymbol from './detectedSymbol';
+import closedSymbol from './closedSymbol';
 
 const TABS = {
   activity: {name: t('Activity'), component: Activity},
 };
 
+function getClosestIndex(data, needle) {
```

Comment:
Can we have some tests/docs for this method? It looks like it's not actually the closest index as implied by the function name but the closest one before or on the target date? Do we not want to return the actual closest one?

Reply:
Added some comments - I think having a close enough point is fine for now, I'll  come back to this later and see if some basic algebra will be enough to find intersection given just an x coordinate.